src/splitInteger.test.js

```diff
@@ -7,13 +7,31 @@ describe(`Function 'splitInteger':`, () => {
     expect(splitInteger).toBeInstanceOf(Function);
   });
 
-  it(`should return an array`, () => {
+  it(`should return an array of numbers`, () => {
+    expect(splitInteger()).toBeInstanceOf(Array);
 
+    for (const result in splitInteger) {
+      expect(typeof result).toEqual(['number']);
+    }
   });
 
   it(`should return an array with 'num' if parts = 1`, () => {
+    expect(splitInteger(5, 1)).toEqual([5]);
+  });
+
+  it(`should  return integer numbers `, () => {
+    expect(splitInteger(7, 2)).toEqual([3, 4]);
+  });
+
+  it(`should return an array that length is equal to the 'parts' value`, () => {
+    expect(splitInteger(5, 5)).toEqual([1, 1, 1, 1, 1]);
+  });
```

Comment:
You can use [toHaveLength](https://jestjs.io/docs/expect#tohavelengthnumber) in this case

Reply:
done